redash/query_runner/athena.py

```diff
@@ -149,7 +149,12 @@ def run_query(self, query, user):
             column_tuples = [(i[0], _TYPE_MAPPINGS.get(i[1], None)) for i in cursor.description]
             columns = self.fetch_columns(column_tuples)
             rows = [dict(zip(([c['name'] for c in columns]), r)) for i, r in enumerate(cursor.fetchall())]
-            data = {'columns': columns, 'rows': rows}
+            qbytes = None
+            try:
+                qbytes = cursor.data_scanned_in_bytes()
+            except AttributeError as e:
+                debug("Athena Direct can't get data_scanned_in_bytes: %s", e)
+            data = { 'columns': columns, 'rows': rows, 'data_scanned': [{ qbytes }] }
```

Comment:
If a string I would imagine something more like this:
`data = { 'columns': columns, 'rows': rows, 'data_scanned': qbytes}`

The `[...]` and `{...}` turn into a set comprehension inside a list comprehension...
```
>>> s = 'test'
>>> {'key1': 'value1', 'key2': [{ s }]}
{'key1': 'value1', 'key2': [set(['test'])]}